examples/pytorch/language-modeling/run_clm.py

```diff
@@ -279,27 +279,30 @@ def main():
     # In distributed training, the load_dataset function guarantee that only one local process can concurrently
     # download the dataset.
     if data_args.dataset_name is not None:
+        kwargs = dict(
+            cache_dir=model_args.cache_dir,
+            use_auth_token=True if model_args.use_auth_token else None,
+        )
+        if data_args.streaming:
+            kwargs.update(streaming=True)
```

Comment:
Let's not refactor kwargs, it makes the example harder to read as we have to go back and forth to see what's in them.